jscomp/bsb_exe/rescript_main.ml

```diff
@@ -84,13 +84,13 @@ let ninja_command_exit (type t) (ninja_args : string array) : t =
    ninja -C _build
 *)
 let clean_usage =
-  "Usage: rescript.exe clean <options>\n\n\
-   `rescript clean` only cleans the current project\n"
+  "Usage: rescript clean <options>\n\n\
```

Comment:
Removed exe and updated the description. It's more relevant now, since the `-with-deps` is deprecated.